install.sh

```diff
@@ -250,9 +251,9 @@ function extract_install () {
     cd "${CREW_DEST_DIR}"
 
     #extract and install
-    echo "Extracting ${1} (this may take a while)..."
+    echo -e "Extracting ${1} (this may take a while)..."
```

Comment:
Missing `${BLUE}` at the start and `${RESET}` at the end of the string?

Reply:
I think the original color (white) is better :)